### DISCUSSION TITLE
Option to use Google Noto Color Emoji as alternative to Twemoji

### DISCUSSION THREAD

**Discussion by ajhalili2006** [squidfunk/mkdocs-material#7684] [Q&A] [answered]
The [Twemoji](https://github.com/twitter/twemoji) repository has not been updated for the past year or so and has not released a new version since March 2022. Do you consider adding support for [Noto Color Emoji](https://fonts.google.com/noto/specimen/Noto+Color+Emoji) for new emojis in [Unicode 16](https://emojipedia.org/unicode-16.0)?

**Comment by facelessuser** [comment, marked as answer]
While the original Twemoji repository has been abandoned, some who were part of the original team, have kept it alive here: https://github.com/jdecked/twemoji. This is what Pymdown Extensions uses and what I assume Material uses as they piggy back off of Pymdown Extensions.

**Reply by squidfunk** [reply]
Yes, we're using the DB managed by Python Markdown Extensions:

https://github.com/squidfunk/mkdocs-material/blob/9ec2ee094cd23cf467051a1d77804853ab3ddfc6/src/extensions/emoji.py#L31

**Reply by ajhalili2006** [reply]
Thanks @squidfunk @facelessuser for the note. Maybe I didn't search around GitHub and chose the FAFO route. Closing this discussion thread to avoid clutter.
